docs/src/xhtml/components/datepicker/index.xhtml

```diff
@@ -35,7 +35,15 @@
 								value: '1984-05-23',
 								min: '1984-01-01',
 								max: '1985-12-24',
+								onrendercell: function(cell){
```

Comment:
this is the example for setting a `min`/`max` value, could you make another separate block focused on the `onrendercell` callback?